units/meson.build

```diff
@@ -13,6 +13,8 @@ units = [
         ['veritysetup-pre.target',              'HAVE_LIBCRYPTSETUP'],
         ['veritysetup.target',                  'HAVE_LIBCRYPTSETUP',
          'sysinit.target.wants/'],
+        ['integritysetup.target',               '',
+         'sysinit.target.wants/'],
```

Comment:
should be conditionalized by HAVE_LIBCRYPTSETUP as above

(and as mentioned, let's add veritysetup-pre.target right away, please